apps/api/pkg/database/tenant.go

```diff
@@ -36,13 +36,13 @@ func GetTenantById(db *sql.DB, id string) (*models.Tenant, error) {
 	return &tenant, nil
 }
 
-func UpdateTenantName(db *sql.DB, id string, newName string) error {
-    query := "UPDATE tenants SET name = $1, updated_at = CURRENT_TIMESTAMP WHERE id = $2"
+func UpdateTenantName(db *sql.DB, id int64, newName string) error {
```

Comment:
Why we convert string id to int64?

Reply:
beacuse our database schema tenantid is integer